docs/content/docs/dev/table/common.md

```diff
@@ -411,6 +411,28 @@ tableEnv.createTemporaryView("`example.View`", table)
 tableEnv.createTemporaryView("other_catalog.other_database.exampleView", table)
 ```
 {{< /tab >}}
+{{< tab "Python" >}}
```

Comment:
Could we also add Python example for the following example: https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/dev/table/common/#connector-tables